app/vmselect/netstorage/netstorage.go

```diff
@@ -1965,10 +1967,10 @@ func startStorageNodesRequest(qt *querytracer.Tracer, sns []*storageNode, denyPa
 	f func(qt *querytracer.Tracer, workerID uint, sn *storageNode) any,
 ) *storageNodesRequest {
 	resultsCh := make(chan rpcResult, len(sns))
-	qts := make(map[*querytracer.Tracer]struct{}, len(sns))
+	qts := make(map[*querytracer.Tracer]string, len(sns))
 	for idx, sn := range sns {
-		qtChild := qt.NewChild("rpc at vmstorage %s", sn.connPool.Addr())
-		qts[qtChild] = struct{}{}
+		qtChild := querytracer.NewOrphan(qt, "rpc at vmstorage %s", sn.connPool.Addr())
```

Comment:
it should be `qtOrphan` and would be great to have a comment why it is created as an orphan
and issue link

Reply:
Added comment with link.